scripts/dev_setup.sh

```diff
@@ -15,10 +15,10 @@
 # fast fail.
 set -eo pipefail
 
-Z3_VERSION=4.8.13
+Z3_VERSION=4.10.1
```

Comment:
Also, Z3 [4.10.2](https://github.com/Z3Prover/z3/releases/tag/z3-4.10.2) is released as well.

Reply:
Done.